src/email.rs

```diff
@@ -12,10 +12,15 @@ use lettre::transport::stub::StubTransport;
 use lettre::{Message, Transport};
 use rand::distributions::{Alphanumeric, DistString};
 
+pub trait Email {
+    const SUBJECT: &'static str;
+    fn body(&self) -> String;
+}
+
 #[derive(Debug, Clone)]
 pub struct Emails {
     backend: EmailBackend,
-    domain: String,
+    pub domain: String,
```

Comment:
questionable, but good enough for now, I guess :D

Reply:
I'll allow it. :laughing: 

Turning this into some sort of `EmailState` type or whatever that gets passed into `body` and can provide the domain etc would probably be a good first issue for someone.

Reply:
I like that idea! 👍